src/transformers/models/swinv2/modeling_tf_swinv2.py

```diff
@@ -0,0 +1,1478 @@
+# coding=utf-8
+# Copyright 2022 Microsoft Research and The HuggingFace Inc. team. All rights reserved.
```

Comment:
In case you don't plan to add a TF implementation, this file needs to be removed from the PR.

Reply:
I will remove this file, will try to add TF implementation in separate PR.